assets/Zed/js/modules/form-toggler.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+'use strict';
+
+$(document).ready(function() {
+
+    createToggleElements();
+
+    $('.toggle-trigger').each(function (index, selectItem) {
+        var group = $(selectItem).data('toggle-group'),
+            type = selectItem.value,
+            items = getItems(group),
+            activeItem = getActiveItem(group, type);
+
+        toggleItems(items, activeItem);
+
+        $(selectItem).on('change', function (event) {
+            type = event.target.value;
```

Comment:
why you call it as type?

Reply:
It's a agreement with BE.

Reply:
I'm fine if you'll find a better name here

Reply:
`value` ?

Reply:
I do not think that `value` is a better name here, 
please see for what reason this select value is used

Reply:
Fixed